unified-codes/libs/ui/src/lib/components/entity/molecules/EntityDetailList.tsx

```diff
@@ -0,0 +1,60 @@
+import * as React from 'react';
+
+import { Property, IEntity } from '@unified-codes/data';
+import { Typography } from '../../data';
+import { Grid } from '../../layout';
+import { Link } from '../../navigation';
+import { EntityDetailCategoryItem } from './EntityDetailCategoryItem';
+import { TypographyVariant } from '@material-ui/core';
+
+export interface IExternalLink {
+  type: string;
+  url: string;
+}
+
+export interface EntityDetailListProps {
+  headerVariant?: TypographyVariant;
+  productSubCategories?: IEntity[];
+  entityProperties?: Property[];
+  externalLinks?: IExternalLink[];
+}
+
+export type EntityDetailList = React.FunctionComponent<EntityDetailListProps>;
+
+export const EntityDetailList: EntityDetailList = ({
+  headerVariant,
+  productSubCategories,
+  externalLinks,
+  entityProperties,
+}) => {
+  const headerTypography = headerVariant ?? 'h6';
+
+  return (
+    <Grid direction="column">
+      <Typography variant={headerTypography}>Forms</Typography>
+      <ul>
+        {productSubCategories?.map((category) => (
+          <EntityDetailCategoryItem entity={category} />
+        ))}
+      </ul>
+      <Typography variant={headerTypography}>Properties</Typography>
```

Comment:
I wonder about not showing a section if there are no elements in it?
Either that or showing 'none' under the heading?

Reply:
I like the idea of showing 'none' under the header (shows there is potentially some stuff but not for that particular product).